### PR TITLE
Resolving #648 (PTC-W0048)

### LINKED ISSUE #648: (PTC-W0048) `if` statements can be merged
## Description
Nested `if` statements can be collapsed into a single `if` statement by separating their condition using `and` operator. Merging collapsible `if` statements increases the code&#x27;s readability.

## Occurrences
There is 1 occurrence of this issue in the repository.

See all occurrences on DeepSource &rarr; [app.deepsource.com/gh/WesternFriend/WF-website/issue/PTC-W0048/occurrences/](https://app.deepsource.com/gh/WesternFriend/WF-website/issue/PTC-W0048/occurrences/)

### PR DESCRIPTION
**Pull request by P-Pole**
Closes #648 

Resolved a minor issue with #648 and merged separate `if` statements.

Before: 

```python
        if request.method == "POST":
            if donation_form.is_valid():
                return process_donation_request(
                    request, donation_form, donor_address_form
``` 
After:
```python
        if request.method == "POST" and donation_form.is_valid():

            return process_donation_request(
                request, donation_form, donor_address_form
```
```diff
-          if request.method == "POST":
-               if donation_form.is_valid():
+         if request.method == "POST" and donation_form.is_valid():
```

### CODE REVIEW

File: donations/models.py [hunk 1]
```diff
@@ -66,10 +66,10 @@ def serve(self, request, *args, **kwargs):
...
         donation_form = DonationForm(request.POST)
 
-        if request.method == "POST":
-            if donation_form.is_valid():
-                return process_donation_request(
-                    request, donation_form, donor_address_form
+        if request.method == "POST" and donation_form.is_valid():
+           
+            return process_donation_request(
+                request, donation_form, donor_address_form
```
Comment (brylie) [inline donations/models.py:72]: Please lint these changes with `pre-commit` or `black`

**Review by brylie — CHANGES_REQUESTED** [review]
Looks great overall. Just a minor detail to run automatic linters on the code. We have a series of `pre-commit` hooks that analyze and fix low-level linting issues as well as running type analysis.

File: donations/models.py (hunk 1 above)
Comment (P-Pole) [reply donations/models.py]: I did use `black` and commited the code again. Please let me know if everything is fine.

**Review by brylie — APPROVED** [review]

**Comment by brylie** [comment]
@all-contributors please add @P-Pole for code

**Pull request merged**